resources/local/environment.go

```diff
@@ -41,3 +52,12 @@ func (e *Environment) InternalDockerhubMirror() string {
 func (e *Environment) InternalRegistryImageTagExists(_, _ string) (bool, error) {
 	return true, nil
 }
+
+// Common
+func (e *Environment) DefaultPublicKeyPath() string {
+	return e.InfraConfig.Get(DDInfraDefaultPublicKeyPath)
+}
+
+func (e *Environment) GetCommonEnvironment() *config.CommonEnvironment {
```

Comment:
Do you really need to access the underlying common environment? I do not expect it to be used since we introduced the config.Env interface

Reply:
Removing this